### ISSUE TITLE
Clicking "Run" in examples menu causes chart iFrame to go blank

### ISSUE THREAD

**Issue by dillonreedy** [recharts/recharts.org#155]
When navigating to the examples, if a user clicks on the "Run" button, the chart will go blank.

**Chart before clicking "Run":**
![image](https://user-images.githubusercontent.com/3454995/103492205-dcfc2780-4dee-11eb-9fdd-7ae0f9ff0840.png)

**Chart after clicking "Run":**
![image](https://user-images.githubusercontent.com/3454995/103492226-00bf6d80-4def-11eb-8770-111d5ae6d9be.png)

**Comment by ckifer** [comment]
This needs fixed - error is 
```javascript
Uncaught TypeError: Cannot read property 'left' of undefined
    at e.renderClipPath (generateCategoricalChart.tsx:1935)
```
The Monaco editor is not working correctly/state is not being managed when we click run here.

**Comment by ckifer** [comment]
should be able to see the charts now after clicking the Run button - it was a height issue but my fix was small and quick - probably needs some follow up TLC

**Comment by Yilun-Sun** [comment]
Solved #239
